Os/Stub/test/DefaultTask.cpp

```diff
@@ -0,0 +1,37 @@
+// ======================================================================
+// \title Os/Posix/DefaultTask.cpp
+// \brief sets default Os::Task to posix implementation via linker
+// ======================================================================
+#include <cerrno>
+#include "Os/Task.hpp"
+#include "Os/Stub/test/Task.hpp"
+#include "Os/Delegate.hpp"
+#include <sys/time.h>
+
+namespace Os {
+    Os::Task::Status TaskInterface::delay(Fw::Time interval) {
+        Os::Stub::Task::Test::StaticData::data.lastCalled = Os::Stub::Task::Test::StaticData::LastFn::DELAY_FN;
+        Os::Stub::Task::Test::StaticData::data.delay = interval;
+
+        timeval start;
+        timeval end;
+        if (gettimeofday(&start, nullptr) == 0) {
```

Comment:
What is the reason for busy-waiting in a unit test stub? This looks unusual. Normally one would busy-wait only in low-level OS or embedded code. What is the intended use case for this busy wait?

Reply:
This is a test implementation of `delay` it just busy waits such that tests see the delay.

Reply:
Added a comment.